pkg/certificates/renewerbottlerocket.go

```diff
@@ -0,0 +1,358 @@
+package certificates
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+const (
+	brEtcdCertDir           = "/var/lib/etcd"
+	brControlPlaneCertDir   = "/var/lib/kubeadm/pki"
+	brControlPlaneManifests = "/var/lib/kubeadm/manifests"
+	brTempDir               = "/run/host-containerd/io.containerd.runtime.v2.task/default/admin/rootfs/tmp"
+)
+
+// BottlerocketRenewer implements OSRenewer for Bottlerocket systems.
+type BottlerocketRenewer struct {
+	osType OSType
+	backup string
+}
+
+// NewBottlerocketRenewer creates a new BottlerocketRenewer.
+func NewBottlerocketRenewer(backupDir string) *BottlerocketRenewer {
+	return &BottlerocketRenewer{
+		osType: OSTypeBottlerocket,
+		backup: backupDir,
+	}
+}
+
+// RenewControlPlaneCerts renews certificates for control plane nodes.
+func (b *BottlerocketRenewer) RenewControlPlaneCerts(
+	ctx context.Context,
+	node string,
+	cfg *RenewalConfig,
+	component string,
+	ssh SSHRunner,
+) error {
+	logger.V(0).Info("Processing control-plane node", "node", node)
+
+	hasExternalEtcd := cfg != nil && len(cfg.Etcd.Nodes) > 0
+
+	if hasExternalEtcd {
+		if err := b.transferCertsToControlPlane(ctx, node, ssh); err != nil {
+			return fmt.Errorf("transferring certificates to control plane node: %v", err)
+		}
+	}
+
+	sessionCmds := buildBRSheltieCmd(
+		buildBRImagePullCmd(),
+		buildBRControlPlaneBackupCertsCmd(component, hasExternalEtcd, b.backup, brControlPlaneCertDir),
+		buildBRControlPlaneRenewCertsCmd(),
+		buildBRControlPlaneCheckCertsCmd(),
+		buildBRControlPlaneCopyCertsFromTmpCmd(),
+		buildBRControlPlaneRestartPodsCmd(),
```

Comment:
Can we change the names to this to make it more readable?

sessionCmds := sheltie(
		pullContainerImage(),
		backupControlPlaneCerts(component, hasExternalEtcd, b.backup, brControlPlaneCertDir),
		renewControlPlaneCerts(),
		checkControlPlaneCerts(),
		copyExternalEtcdCerts(),
		restartControlPlaneStaticPods(),

Can you pl change these names for Ubuntu too in the other PR that you have?